src/avatar/Avatar.js

```diff
@@ -77,9 +77,6 @@ const AvatarComponent = ({
       />
     ));
 
-  // Remove placeholder styling if we're not using image
```

Comment:
If source and source.uri both exist then only hide the placeholder.

Reply:
okay